### DISCUSSION TITLE
Help us collecting a list of community projects made with NiceGUI

### DISCUSSION THREAD

**Discussion by Diegiwg** [zauberzeug/nicegui#396] [General]
> Is there any movement by you (nicegui) to create a space for the dissemination of projects that use the lib?

**Comment by rodja** [comment]
Year! That would be really great. Maybe we should directly list the on https://nicegui.io. Let's start collecting a list here and if some projects are added we will place them accordingly. After that, pull requests will be fairly easy to integrate new projects.

**Comment by rodja** [comment]
We have created a GitHub wiki with a section for [Community Projects](https://github.com/zauberzeug/nicegui/wiki#community-projects). Help us filling in all the great projects you have done or found on the internet.
